src/org/labkey/test/components/react/MultiMenu.java

```diff
@@ -40,48 +40,73 @@ private WebElement getMenuList()
         return Locator.tagWithClass("ul", "dropdown-menu").findElement(this);
     }
 
+    /*
+        finds the first menu item with the specified text
+     */
     public WebElement getMenuItem(String text)
     {
-        return Locator.tag("li").withChild(Locator.tagWithAttribute("a", "role", "menuitem")
-                    .withText(text)).waitForElement(getMenuList(), _menuWaitTmeout);
+        return Locators.menuItem().withText(text).waitForElement(getMenuList(), _menuWaitTmeout);
     }
 
+    /**
+     * Find the first item with specified text under the path of the
+     * @param menuText Text of the menu to expand
+     * @param itemText  Text of the item to find under its path
+     * @return
+     */
+    public WebElement getMenuItem(String menuText, String itemText)
+    {
+        return getItemUnderToggle(menuText, itemText);
+    }
```

Comment:
Why not just make `getItemUnderToggle` public? I like that name better because it is more explicit; maybe name it `getMenuItemUnderToggle` to match the name of `getMenuItem`.

Reply:
The only reason for that, really, was to align to the convention where the parameters of like-named methods with 1 and 2 params signify that the second one, if any, is to be found under a menu specified by the first.
But, I'll take this question as a suggestion